File: app/src/main/java/pedroluiz/projeto/soccernews/data/local/SoccerNewsDb.java
package pedroluiz.projeto.soccernews.data.local;

import androidx.room.Database;
import androidx.room.RoomDatabase;

import pedroluiz.projeto.soccernews.domain.model.News;

@Database(entities = {News.class}, version = 1)
public abstract class SoccerNewsDb extends RoomDatabase {

    public abstract NewsDao newsDao();

}

File: app/src/main/java/pedroluiz/projeto/soccernews/data/local/SoccerNewsDb.kt
package pedroluiz.projeto.soccernews.data.local

import androidx.room.Database
import androidx.room.RoomDatabase
import pedroluiz.projeto.soccernews.domain.News

@Database(entities = [News::class], version = 1)
abstract class SoccerNewsDb : RoomDatabase() {
    abstract fun newsDao(): NewsDao?
}